test/serialization.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I think instead of creating unused variables to filter object fields with destructuring, you should create a helper function to do that.
For example:
```js
const omitKeys = (obj, keys) => {
  const res = {};
  for (const key in obj) {
    if (!keys.includes(key)) res[key] = obj[key];
  }
  return res;
};
```